src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -265,7 +277,22 @@
                                                         (fn [input-state]
                                                           (controlled-input/set-input-value
                                                            input-state
-                                                           max-value)))))]
+                                                           max-value)))))
+        on-refresh-swap-proposal                    (fn []
+                                                      (when (and valid-pay-input?
+                                                                 (not loading-swap-proposal?))
+                                                        (fetch-swap-proposal
+                                                         {:amount       pay-input-amount
+                                                          :valid-input? valid-pay-input?})))]
```

Comment:
Just throwing a suggestion to clear the interval if the input is not valid.

Reply:
Thanks for the suggestion, updated the code to clear the interval in case `valid-pay-input?` is false